Week-4/HangMan.py

```diff
@@ -4,26 +4,39 @@
 class HangMan:
     def __init__(self, word: str = "BOLLYWOOD") -> None:
         self.__word = word.upper()
-        self.__word_letters = {char for char in self.__word}
+        self.__letter_left = {char for char in self.__word}
```

Comment:
why are we using __ , because generally __ is used for internal python methods for example if you want to implement + operaror you will implement __add__, same goes for __str__